SingularityUI/app/assets/index.mustache

```diff
@@ -45,8 +51,14 @@
             globalRefreshInterval: 60000
         };
     </script>
+    {{#isDebug}}
+    <script src="/static/js/app.bundle.js"></script>
+    <script src="/static/js/vendor.bundle.js"></script>
+    {{/isDebug}}
+    {{^isDebug}}
+    <script src="{{{staticRoot}}}/js/app.bundle.js"></script>
     <script src="{{{staticRoot}}}/js/vendor.bundle.js"></script>
-    <script src="{{{staticRoot}}}/js/app.js"></script>
+    {{/isDebug}}
```

Comment:
Gulp drops the index.html file into the dist folder, but not in the static folder. I mostly copied the way it worked before, but I would be interested in changing things there if we are interested.